src/expr/sequence.h

```diff
@@ -0,0 +1,168 @@
+/*********************                                                        */
+/*! \file sequence.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief The sequence data type.
+ **/
+
+#ifndef CVC4__EXPR__SEQUENCE_H
+#define CVC4__EXPR__SEQUENCE_H
+
+#include <vector>
+#include "expr/node.h"
+
+namespace CVC4 {
+
+class ExprSequence;
```

Comment:
```suggestion
```

Reply:
This is in preparation for the expr-level sequence data type, to be added next.